package.json

```diff
@@ -48,6 +48,7 @@
     "vue": "^2.7.10",
     "vue-affix": "^0.5.2",
     "vue-debounce-decorator": "^1.0.1",
+    "vue-hotjar": "^1.4.0",
```

Comment:
For future use.

I already configured HOTJAR_ID in 1Password (for each environment), which gets built into a config map during deployment, and which this app fetches as "configuration.json".

This is for future use.